File: C/bee 1036.c
#include <stdio.h>
#include <math.h>

int main(void) {

    double a, b, c;
    scanf("%lf %lf %lf", &a, &b, &c);
    double delta = pow(b,2) - (4 * a * c);

    if (a != 0 && delta >= 0) {
        double R1 = (-b + sqrt(delta))/(2*a);
        double R2 = (-b - sqrt(delta))/(2*a);

        printf("R1 = %.5lf\nR2 = %.5lf\n", R1, R2);
    } else {
        printf("Impossivel calcular\n");
    }



    return 0;
}

File: Lua/bee 1036.lua
local a = tonumber(io.read("*n"))
local b = tonumber(io.read("*n"))
local c = tonumber(io.read("*n"))
local delta = b^2 - (4 * a * c)

if a ~= 0 and delta >= 0 then
    local R1 = (-b + math.sqrt(delta))/(2*a)
    local R2 = (-b - math.sqrt(delta))/(2*a)
    print(string.format("R1 = %.5f\nR2 = %.5f", R1, R2))
else
    print("Impossivel calcular")
end
